tools/clinReport/clinReport.def

```diff
@@ -19,6 +19,8 @@ From: ubuntu:18.04
 
     cd clin-report
 
+    git checkout 1c67fcf
```

Comment:
Can you use the full SHA1 hash here? In this case it's 1c67fcffc17e803ce1c1b41aff34470547ffa48c

Reply:
Actually it is 1da9753dc40903afb1604094514f2f59b4dd122e for the latest commit. I will make the update.